og.module

```diff
@@ -210,12 +210,17 @@ function og_entity_create_access(AccountInterface $account, array $context, $bun
       continue;
     }
 
-    $handler = Og::getSelectionHandler($field_definition);
-
-    if ($handler->getReferenceableEntities()) {
+    $options = [
+      'target_type' => $field_definition->getFieldStorageDefinition()->getSetting('target_type'),
+      'handler' => $field_definition->getSetting('handler'),
+      'field_mode' => 'admin',
```

Comment:
Did we introduce a bug here, before this PR there wasn't a 'field_mode' => 'admin' in the config?

cc @pfrenssen, @kbrodej